### PR TITLE
STRM-764 remove sink commands and add data connector commands

### PR DESCRIPTION
**Pull request by ivan-p92**
This PR:
* removes all sink-related commands
* makes sure the data connector api and/or ref is used instead of the sink api / ref where applicable
* adds commands for the data connector api
* updates any textual references to the concept of sinks

### CODE REVIEW

File: pkg/entity/data_connector/AzureBlobStorage.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
+package data_connector
```
Comment (trietsch) [inline pkg/entity/data_connector/AzureBlobStorage.go:1]: Please check the styleguide of Golang. I think file names should be snake cased.

File: pkg/entity/data_connector/AzureBlobStorage.go [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+	tenantIdFlag          = "tenant-id"
+	clientIdFlag          = "client-id"
+	clientSecretFlag      = "client-secret"
+)
+
+func createAzureBlobStorageCmd() *cobra.Command {
+	azureBlobStorage := &cobra.Command{
+		Use:               "azure-blob-storage [data-connector-name] [container-name]",
+		Short:             "Create a Data Connector for an Azure Blob Storage Container",
+		Long:              longDoc,
```
Comment (trietsch) [inline pkg/entity/data_connector/AzureBlobStorage.go:20]: Missing `longDoc` here?

File: pkg/entity/data_connector/gcsBucket.go [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+					},
+				},
+			}
+			create(dataConnector)
+		},
+		Args: cobra.ExactArgs(2),
+	}
+
+	flags := gcsBucket.Flags()
+	flags.String(credentialsFileFlag, "", "file with bucket credentials json")
```
Comment (trietsch) [inline pkg/entity/data_connector/gcsBucket.go:37]: `file with bucket credentials json` -> `file with service account JSON key credentials`

File: pkg/entity/data_connector/s3Bucket.go [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+					},
+				},
+			}
+			create(dataConnector)
+		},
+		Args: cobra.ExactArgs(2),
+	}
+
+	flags := s3Bucket.Flags()
+	flags.String(credentialsFileFlag, "", "file with bucket credentials json")
```
Comment (trietsch) [inline pkg/entity/data_connector/s3Bucket.go:41]: Maybe something more descriptive to the specification of this format? IIRC it's a format defined by AWS themselves.

**Review by trietsch — CHANGES_REQUESTED** [review]

File: pkg/entity/data_connector/AzureBlobStorage.go (hunk 2 above)
Comment (ivan-p92) [reply pkg/entity/data_connector/AzureBlobStorage.go:20]: Ah, all specific connectors currently use the same long doc as the parent data-connector command. Probably better to make them specific.

**Comment by trietsch** [comment]
Includes changes from https://github.com/strmprivacy/cli/pull/89

**Pull request merged**